README.md

```diff
@@ -37,8 +41,7 @@ Instead, it will copy all the configuration files and the transitive dependencie
 
 You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
 
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+## Developers
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+### Asana Integration
```

Comment:
Thanks for the extra instructions, here!

Reply:
No worries. They are still not ideal but I figured for this context it would be enough to get it working on your machine. Ideally I would reconfigure things so that starting it up with the backend is still one command